### PR TITLE
September Edge Roundup

### PR DESCRIPTION
**Pull request by kflynn**
Signed-off-by: Flynn <flynn@buoyant.io>

### CODE REVIEW

**Comment by kflynn** [comment]
cc @CathPag

**Review by CathPag — APPROVED** [review]
LGTM

File: linkerd.io/content/blog/2024/0904-edge-release-roundup.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+## Recommendations and breaking changes
+
+All these releases are recommended for general use. Happily, there are no
+breaking changes here.
+
+## The releases
+
+August's edge releases look small, but [`edge-24.8.1`] and [`edge-24.8.2`]
+provided the finishing touches for [Linkerd 2.16.0], which shipped on August
```
Comment (wmorgan) [inline linkerd.io/content/blog/2024/0904-edge-release-roundup.md:56]:
```suggestion
provided the finishing touches for [Linkerd 2.16], which shipped on August
```

**Review by wmorgan — APPROVED** [review]

File: linkerd.io/content/blog/2024/0904-edge-release-roundup.md (hunk 1 above)
Comment (wmorgan) [reply linkerd.io/content/blog/2024/0904-edge-release-roundup.md:56]: Oops, forgot to also update the corresponding link later in the file.

**Pull request merged**